landscape.yml

```diff
@@ -12111,6 +12128,13 @@ landscape:
             homepage_url: https://elastx.se/en/kubernetes/
             logo: elastx.svg
             crunchbase: https://www.crunchbase.com/organization/elastx
+            name: Enterprise DB (KCSP)
```

Comment:
This new entry needs a new item. It's clashing with the entry above.

Add `- item:` above line 12131